sagemaker-lineage/sagemaker-lineage.ipynb

```diff
@@ -159,6 +159,8 @@
    "outputs": [],
    "source": [
     "# create an artifact representing inputs to the model building action\n",
+    "# Note that the CreateArtifact API is restricted to SageMaker only.  For more information please see CreateArtifact.\n",
```

Comment:
Under "For more information please see CreateArtifact.", it will be good if we can redirect to "CreateArtifact"?

Reply:
good idea, added